### DISCUSSION TITLE
Error!!

### DISCUSSION THREAD

**Discussion by hanselsaji** [TobiasPankner/Teams-Auto-Joiner#143] [Q&A] [answered]
After I type python auto-start.py
It opens chrome, and log in to my account, and automatically leaves.
Saying the error was "Timeout Error"

**Comment by TobiasPankner** [comment, marked as answer]
Create an issue and post the full log and config please.

**Reply by hanselsaji** [reply]
Sure

**Comment by hanselsaji** [comment]
Sure!   From: TobiasPanknerSent: Thursday, September 16, 2021 1:25 PMTo: TobiasPankner/Teams-Auto-JoinerCc: hanselsaji; AuthorSubject: Re: [TobiasPankner/Teams-Auto-Joiner] Error!! (Discussion #143) Create an issue and post the full log and config please.—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android.  

	
        
		Virus-free. www.avast.com
